cpu/cc2538/periph/timer.c

```diff
@@ -27,43 +27,35 @@
 #include "periph/timer.h"
 #include "periph_conf.h"
 
+#include "vendor/hw_gptimer.h"
+#include "vendor/hw_memmap.h"
+#include "cc2538_gptimer.h"
+#include "cc2538_sys_ctrl.h"
+
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
-#define LOAD_VALUE               (0xffff)
-
-#define TIMER_A_IRQ_MASK         (0x000000ff)
-#define TIMER_B_IRQ_MASK         (0x0000ff00)
-
-#define BIT(n)                   ( 1UL << (n) )
+#define LOAD_VALUE              (0xffff)
 
-/* GPTIMER_CTL Bits: */
-#define TBEN                     BIT(8)
-#define TAEN                     BIT(0)
+#define TIMER_A_IRQ_MASK        (0x000000ff)
+#define TIMER_B_IRQ_MASK        (0x0000ff00)
 
-/* GPTIMER_TnMR Bits: */
-#define TnCMIE                   BIT(5)
-#define TnCDIR                   BIT(4)
+/* GPTIMER_CTL Bits */
+#define TBEN                    GPTIMER_CTL_TBEN
+#define TAEN                    GPTIMER_CTL_TAEN
 
-/* GPTIMER_IMR Bits: */
-#define TBMIM                    BIT(11)
-#define TAMIM                    BIT(4)
+/* GPTIMER_TnMR Bits */
+#define TnMIE                   GPTIMER_TAMR_TAMIE
```

Comment:
Why CamelCase?

Reply:
good question, ah copy-paste from before its either T**A**MIE or T**B**MIE, should I change to upper case only?

Reply:
Ah I see. Yes, let's take upper case only